packages/components/organisms/f-checkout/src/components/TermsAndConditions.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div
+        data-test-id="checkout-terms-and-conditions-component"
+        :class="$style['c-checkoutTermsAndConditions']">
+        <i18n
+            path="checkoutTermsAndConditions">
+            <template #termsAndConditions>
+                <a :href="$t('termsAndConditionsLinkUrl')">
```

Comment:
Should these links (including the ones below) open in new tabs seen as they're on a checkout page?